includes/RequestWiki/Specials/SpecialRequestWiki.php

```diff
@@ -280,11 +280,19 @@ public function onSubmit( array $data ): Status {
 		return Status::newGood();
 	}
 
-	public function isValidReason( ?string $reason ): bool|Message {
+	public function isValidReason( ?string $reason ): bool|string|Message {
 		if ( !$reason || ctype_space( $reason ) ) {
 			return $this->msg( 'htmlform-required' );
 		}
 
+		$minLength = $this->getConfig()->get( ConfigNames::RequestWikiMinimumLength );
+		if ( $minLength && strlen( $reason ) < $minLength ) {
+			return $this->msg( 'requestwiki-error-minlength' )->numParams(
+				$minLength,
+				strlen( $reason )
+			)->parse();
```

Comment:
```suggestion
			);
```

Calling `Message::parse()` is not necessary

Reply:
I was aware it did it automatically but thought it would be better to make it explicit for clarity but maybe we don't need to yeah... we'll see...

Reply:
Added a comment instead